### PR TITLE
Performance audit

### PR DESCRIPTION
**Pull request by preciousuwhubetine**
In this branch, @Tafloninno and I worked to optimize database performance by creating new table indices to reduce database query times. Here are the screenshots before and after optimization:

**Before**
performance_1
performance_2
performance_3

**After**
performance4
performance5
performance6

### CODE REVIEW

**Review by bobb-Rob — COMMENTED** [review]
Hi @preciousbetine, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that lacks `commit` from your coding partners. This is a pair programming activity and it is expected that all partners have made significant contributions to the project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: schema.sql [hunk 1]
```diff
@@ -57,3 +57,9 @@ CREATE TABLE visits (
     date_of_visit date,
     PRIMARY KEY(id)
 );
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animal_id_asc ON visits(animal_id ASC);
+CREATE INDEX vet_id_asc ON visits(vet_id ASC);
+CREATE INDEX email_asc ON owners(email ASC);
```
Comment (Zuhaib042) [inline schema.sql:65]: - [x] The code is absolutely fine for adding indexes, but somehow two of queries are not improving. one's result is almost identical before and after, and the second one is actually got worse after indexing so there is something wrong. kindly consider approving these two of your queries.
you can apply this code for indexing 
```
   CREATE INDEX ON visits (animal_id);
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);
```
Also sometimes if you ran same query after indexing, it gets better after first tries, so you can try that also.
your last two queries screen shots before and after
before 
qw

after
a

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @preciousbetine @Tafloninno 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Correct Gitflow
✅ Pair-Programming

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (preciousuwhubetine) [reply schema.sql:65]: @Zuhaib042 Thanks for the review! I noticed that after indexing, I even got worse query results in some cases. Could not figure out why. Had to drop the whole database to get it working!

**Review by Davidosky007 — APPROVED** [review]
Hi @preciousbetine 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* Queries written properly 👍 
* Good looking readme 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**